source/containers.tex

```diff
@@ -9112,14 +9115,14 @@
 \end{codeblock}
 \end{itemdescr}
 
-\rSec2[vector.bool]{Class \tcode{vector<bool>}}
+\rSec2[vector.bool]{Specialization for \tcode{bool}}
```

Comment:
I think `vector` should be mentioned in the subclause title. [vector.bool] is a sibling of [vector], so this is unlike this [specification guideline](https://github.com/cplusplus/draft/wiki/Specification-Style-Guidelines#subclause-headings):
>Do not repeat parts of subclause headings in nested headings (decided in #1242). Good: "Class template tuple" / "Construction". Bad: "Class template vector" / "vector capacity".

This is how it currently looks:
![1661038571](https://user-images.githubusercontent.com/21071787/185769464-a9075492-031b-4681-a2cb-f593a4f6b35e.png)


Bettter precedent:
![1661038592](https://user-images.githubusercontent.com/21071787/185769466-fde92f2b-d9ec-4a1f-87a3-68b2cfdaf133.png)
![1661038610](https://user-images.githubusercontent.com/21071787/185769468-1da19610-39f4-4506-89a9-2b7aa2784e2d.png)


```suggestion
\rSec2[vector.bool]{Specialization for \tcode{vector} of \tcode{bool}}
```

Reply:
Yes, I already had that but forgot to push it, thanks!